### DISCUSSION TITLE
How can I debug issues with the network proxy?

### DISCUSSION THREAD

**Discussion by neugartf** [mobile-dev-inc/Maestro#666] [Q&A] [answered]
Hey,

I'm trying to get some network traffic recorded with `maestro network record [folder]` , but so far it just blocks any outgoing traffic (i.e. the app under inspection is unusable). The certificate is installed and correctly setup in the `network_config.xml`.

What would be the best way to debug this issue? Is there a verbose mode that I haven't found yet?
Should I build everything by my own and then inspect with breakpoints?

Thanks for any help :)!

**Comment by axelniklasson** [comment, marked as answer]
Hi @neugartf! The experimental network mocking feature has unfortunately been deprecated as we are focusing on other more impactful features, but thanks for trying it out!
